openghg/store/base/_base.py

```diff
@@ -233,26 +237,31 @@ def _standardise_from_file(
             filters=filters,
         )
 
+        if filepath is not None:
+            filepaths = normalise_to_filepath_list(filepath)
+        else:
+            filepaths = []
         for x in datasource_uuids:
-            if isinstance(filepath, list) and len(filepath) == 1:
-                filepath = filepath[0]
-
-            if isinstance(filepath, Path):
-                x.update({"file": filepath.name})
-                logger.info(f"Completed processing: {filepath.name}.")
-            elif isinstance(filepath, list):
-                filepath_str = ", ".join([fp.name for fp in filepath])
+            if not filepaths:
+                continue
```

Comment:
May be better to include `break`? Or include this check outside the loop instead?